config/app_config.py

```diff
@@ -43,7 +43,7 @@ def get_app_version():
         "app_deviceflow_clientsecret": os.getenv("APP_DEVICEFLOW_CLIENTSECRET", ""),
         "app_deviceflow_appname": os.getenv("APP_DEVICEFLOW_APPNAME", ""),
         "app_ins_fhir_clientid": os.getenv("APP_INS_FHIR_CLIENTID", "0oa3gvkmdRb5n52e91d6"),
-        "app_ins_fhir_api_base": os.getenv("APP_INS_FHIR_API_BASE", "https://mpwnxqqpp2.execute-api.us-east-1.amazonaws.com/dev")
+        "app_ins_fhir_api_base": os.getenv("APP_INS_FHIR_API_BASE", "https://7zj8l72c27.execute-api.us-east-1.amazonaws.com/prd")
```

Comment:
@dancinnamon-okta So you are expecting the DEFAULT to point production on the localhost?